Rock & Roll with Ember.js Octane Edition, State of Frontend and ember-meta (#706)
ember-learn/ember-blog: source/2020-07-11-the-ember-times-issue-155.md

---

## [Rock & Roll with Ember.js Octane edition is here 🔥🐹](https://twitter.com/baaz/status/1276104785079472130)

There's an updated version of [Balint Erdi's (@balinterdi)](https://github.com/balinterdi) excellent book Rock & Roll with Ember.js available, now with coverage for Ember's Octane Edition! 🎉

> If you have not had a chance to give Rock & Roll with Ember.js a read, you should. This is one of the most well-written books on Ember.js. It's a great way to introduce people new to Ember to the framework's conventions and teach them how to write idiomatic Ember code. 

Reviewer: **optional:**

I was thinking that the second sentence could be simpler. I wasn't sure if there's difference between conventions and idiomatic code, so I suggested removing the latter.

```suggestion
If you have not had a chance to give Rock & Roll with Ember.js a read, you should. This is one of the most well-written books on Ember.js. It's a great way to introduce people who are new to Ember to the framework's conventions. 
```

> Using a pragmatic approach of learning by building, Rock & Roll with Ember.js takes the reader through the process of building an Ember application from the ground up. Readers should expect to finish the book with a solid understanding of the fundamentals involved in building Ember applications.

Reviewer: Maybe continue with the pronoun "you"?
```suggestion
Using a pragmatic approach of learn-by-building, Rock & Roll with Ember.js takes you through the process of building an Ember application from the ground up. You can expect to finish the book with a solid understanding of the fundamentals involved in building Ember applications.
```

You can find the book [here](https://www.balinterdi.com/rock-and-roll-with-emberjs/).  

---

## [State of Frontend Survey 📊](https://tsh.io/state-of-frontend/)

> Interested in helping Ember.js 🔥 get the recognition it deserves among other frontend tools? You may want to participate in the State of Frontend Survey.

Reviewer: ```suggestion
Interested in helping Ember.js 🔥 get the recognition it deserves among other frontend tools? You may want to participate in the [State of Frontend Survey](https://tsh.io/state-of-frontend/).
```

<!--alex ignore easy of-course-->

> This survey aims to gather information on the reality of frontend developers' work. It gives us a chance to provide input on tooling we use (Ember of course!). 

Reviewer: **optional:**

For brevity, I was thinking that we could say "assess." Maybe you had a particular meaning that you wanted to convey with "gather information on the reality"?
```suggestion
This survey aims to assess frontend developers' work. It gives us a chance to provide input on tooling that we use (Ember, of course!). 
```

It also allows participants the opportunity to comment on tooling they would like to learn and provide thoughts about the future of frontend development trends.

> The survey should only take 7 minutes or so of your time and could help promote the visibility of Ember.js and work to ensure the survey provides accurate information about what tools developers are using. 

Reviewer: I think breaking the sentence into a few will help here.

```suggestion
The survey should only take 7 minutes or so of your time. Please help promote the visibility of Ember.js. Your participation can better ensure that the survey provides accurate information about what tools developers use.
```

Take a moment and fill out the survey [here](https://tsh.io/state-of-frontend/).

---

## [ember-meta 1.0.0 release 🚢](https://twitter.com/shipshapecode/status/1281406393577283587)

`ember-meta 1.0.0` has been released by [Ship Shape (@shipshapecode)](https://github.com/shipshapecode)! Thanks to [Chris Manson (@mansona)](https://github.com/mansona) for his continued help with this addon 🎉!

> In case you weren't aware `ember-meta` is an addon that helps setup meta for your Prember/Ember blog to support opengraph, microdata, Facebook, Twitter, Slack, etc.

Reviewer: Wasn't sure if you listed Facebook separately from Open Graph for a reason (I don't know too much about `<meta>` tags). In case it was a duplicate, I removed it in the suggestion below.

```suggestion
In case you weren't aware, `ember-meta` is an addon that helps set up `<meta>` tags for your Prember/Ember blog. These tags can support Open Graph, microdata, Twitter, Slack, etc.
```

> This release updates dependencies, makes some conversions to native classes, and now allows overloading some config.

Reviewer: ```suggestion
This release updates dependencies, uses native class for the underlying service, and allows overloading some configurations.
```
